reddit_decider/__init__.py

```diff
@@ -560,10 +560,20 @@ def get_variant_for_identifier_without_expose(
         return variant
 
     def get_all_variants_without_expose(self) -> Dict[str, Optional[str]]:
-        """Return a dict of experiment name strings as keys and
-            variant names as the values. If a variant was `None` for an experiment,
-            that experiment is not included in the return dict.
-            All available experiments get bucketed. Exposure events are not emitted.
+        """Return a list of experiment dicts in this format:
+                [
+                    {
+                        "id": "1",
```

Comment:
Is the ID str or int from https://github.com/reddit/experiments.py/blob/develop/reddit_experiments/__init__.py#L127 ?

Reply:
in this case, I'm just making it match the final form of the gql response, where all values are strings.